src/renderers/WebGLRenderer.js

```diff
@@ -1165,15 +1172,6 @@ class WebGLRenderer {
 
 			if ( this.info.autoReset === true ) this.info.reset();
 
-
-			//
-
-			if ( xr.enabled === false || xr.isPresenting === false || xr.hasDepthSensing() === false ) {
```

Comment:
Why is it necessary to move this code block further up?

Reply:
It isn't strictly necessary to move it up, but it felt more at place as now the entire render list setup is in one place: projecting objects, sorting, inserting background elements.

I'm fine either way, so let me know if you'd rather have it back at its old spot.

Reply:
This motivation makes sense to me. We can leave it as it is.